action.yaml

```diff
@@ -11,6 +11,12 @@ inputs:
       are not supported, because partial cache restoration leads to a "snowball"
       effect.
     required: true
+  included-images:
+    description: >
+      If "all", all images will be included.
+      Otherwise, specify precisely all needed images to be included separated with spaces.
+    required: true
```

Comment:
I may be missing something, but I think this should be `false` since, when not specified, the default behavior ought to be the behavior that predates this PR.